care/facility/models/asset.py

```diff
@@ -83,7 +84,10 @@ class Asset(BaseModel):
         choices=AssetTypeChoices, default=AssetType.INTERNAL.value
     )
     asset_class = models.CharField(
-        choices=AssetClassChoices, default=None, null=True, blank=True, max_length=20
+        max_length=20,
+        blank=True,
+        null=True,
+        default=None,
```

Comment:
can't make it choice field since we want our choices to be dynamic. And we are handling the cases to invalidate invalid asset choices through serializers and save method